config/config.yaml

```diff
@@ -49,7 +49,9 @@ tree:
   service: node1.neofs:8080
 
 # RPC endpoint and order of resolving of bucket names
-rpc_endpoint: http://morph-chain.neofs.devenv:30333
+morph:
```

Comment:
Let's call this `fschain` immediately. We will eventually remove `morph` from configs like this.

Reply:
Renamed